tools/public_api_guard/cdk/a11y.d.ts

```diff
@@ -70,6 +70,10 @@ export declare class ConfigurableFocusTrap extends FocusTrap implements ManagedF
     destroy(): void;
 }
 
+export declare class ConfigurableFocusTrapConfig {
```

Comment:
Since this isn't a public API yet, maybe we should reconsider turning it into an `interface`? I don't see a good reason for it to be a `class`.

Reply:
Agreed.